exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js

```diff
@@ -43,14 +44,14 @@ module.exports = function() {
 
 
 
-  this.When(/^receiving the message "([^"]*)"$/, function(message) {
+  When(/^receiving the message "([^"]*)"$/, function(message) {
     this.exocom.send({ service: '_____serviceRole_____',
                        name: message })
   })
 
 
-  this.When(/^receiving the message "([^"]*)" with the payload:$/, function(message, payload, done) {
-    this.fillIn_____modelName@camelcase_____Ids(payload, (filledPayload) => {
+  When(/^receiving the message "([^"]*)" with the payload:$/, function(message, payload, done) {
+    this.fillIn_____modelNameCamelcase_____Ids(payload, (filledPayload) => {
```

Comment:
This is what [tmplconv](https://github.com/okunishinishi/node-tmplconv) uses to render templates right? Why was this changed?